suites/message/nested.go

```diff
@@ -363,7 +352,8 @@ func (s *msStage) send(to address.Address, value abi.TokenAmount, method abi.Met
 		Params: buf.Bytes(),
 	}
 	msg := s.driver.MessageProducer.MultisigPropose(s.msAddr, s.creator, &pparams, chain.Nonce(approverNonce))
-	return s.driver.ApplyMessage(msg)
+	expRet := typegen.CborInt(0)
+	return s.driver.ApplyExpect(msg, chain.MustSerialize(&expRet))
```

Comment:
Multisig propose returns a cbor-int corresponding to the transaction ID, expect that here.

Reply:
Please instead expose something that just returns the value, instead of expecting it.